content/posts/css-component-stack.md

```diff
@@ -0,0 +1,36 @@
+---
+title: component stack con css
+date: "1-10-23"
+description: compo der realizar con componente stack con solo  css
+tags:
+  - css
+---
+
+# Problemática:
+
+cuando trabajamos con htm y css, en la creación layout siempre surge la necesidad de la generación de espacios entre los elementos anteriores o siguientes, esto se puede solucionar muy fácilmente con la propiedad margin, el problema surge cuando estos espacios son agregados en los componentes hijos limitando la reutilización de dichos componentes
```

Comment:
Change **cuando** for **Cuando** tambien **htm** for **html**.

Also finish the text with an **.**